traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -1266,15 +1266,19 @@ sub check_plugins {
 			( my @parts ) = split( /\@plugin\=/, $liner );
 			foreach my $i ( 1..$#parts ) {
 				( my $plugin_name, my $plugin_config_file ) = split( /\@pparam\=/, $parts[$i] );
-				if (defined( $plugin_config_file ) ) {
+				if (defined( $plugin_config_file ) ) {{
```

Comment:
looking again -- that block isn't necessary -- the `last` will exit the nearest enclosing loop,  which is the `foreach ... $#parts)`